src/components/forms/ProxyDeviceForm.tsx

```diff
@@ -40,6 +41,7 @@ interface Props {
 
 const ProxyDeviceForm: FC<Props> = ({ formik, project }) => {
   const notify = useNotify();
+  const [isInstanceBound, setInstanceBound] = useState(false);
```

Comment:
We already store this info in formik, we should avoid shadowing the state here. Rely on the respective formik value instead.